### PR TITLE
Add Royal Roads

### PR DESCRIPTION
**Pull request by stephengroat**
#1744

### CODE REVIEW

File: _data/education.yml [hunk 1]
```diff
@@ -96,6 +96,12 @@ websites:
       img: rmu.png
       tfa: No
 
+    - name: Royal Roads University
+      url: http://www.royalroads.ca
+      twitter: royalroads
```
Comment (Carlgo11) [inline _data/education.yml]: uppercase `RoyalRoads`.
When that's fixed then the PR is good to go :)

**Comment by stephengroat** [comment]
I was at la computadora so that was easy to do

**Comment by Carlgo11** [comment]
Thank you @stephengroat!
Sorry for the late response. It looks like everyone in the core team have been busy lately.

I will go over most of your PRs right now. Hopefully we can merge a few right away :wink:

**Comment by stephengroat** [comment]
@Carlgo11 no problem. hopefully this is done for fun and people aren't threatened with medieval-style torture if they don't approve x sites per week/day/hour :)

**Comment by Carlgo11** [comment]
No! We don't torture people! <b>*cough* *cough*</b>

**Pull request merged**